erpnext/accounts/doctype/bank_reconciliation_tool/bank_reconciliation_tool.js

```diff
@@ -12,6 +12,9 @@ frappe.ui.form.on("Bank Reconciliation Tool", {
 				},
 			};
 		});
+		var no_bank_transactions_text = 
+			`<div class="text-muted text-center">${__("No Matching Bank Transactions Found")}</div>`
+		set_field_options("no_bank_transactions", no_bank_transactions_text);
```

Comment:
Load the text this way make it translatable, as used on:

https://github.com/frappe/erpnext/blob/183662c0e20f65b80dbf459d0fb14fbb8debf997/erpnext/accounts/doctype/loyalty_program/loyalty_program.js#L37